### PR TITLE
Challan amount setting by billing service.

### PR DESCRIPTION
**Pull request by rahu-eGov**

### CODE REVIEW

File: municipal-services/echallan-services/src/main/java/org/egov/echallan/config/ChallanConfiguration.java [hunk 1]
```diff
@@ -259,6 +259,12 @@ public MappingJackson2HttpMessageConverter jacksonConverter(ObjectMapper objectM
 
 	@Value("${egov.today.collection.link}")
 	private String todayCollectionLink;
-	
-	
+
+    @Value("${mgramseva.host}")
+    private String mgramsevaHost;
```
Comment (rushang7-eGov) [inline municipal-services/echallan-services/src/main/java/org/egov/echallan/config/ChallanConfiguration.java:264]: Write this as billing service host.

File: municipal-services/echallan-services/src/main/java/org/egov/echallan/config/ChallanConfiguration.java [hunk 2]
```diff
@@ -259,6 +259,12 @@ public MappingJackson2HttpMessageConverter jacksonConverter(ObjectMapper objectM
...
+
+    @Value("${mgramseva.billing.service.search.bill.endpoint}")
+    private String mgramsevaBillingServiceSearchBillEndpoint;
```
Comment (rushang7-eGov) [inline municipal-services/echallan-services/src/main/java/org/egov/echallan/config/ChallanConfiguration.java:267]: As this is part of mgramseva, no need to mention `mgramseva` prefix.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: municipal-services/echallan-services/src/main/java/org/egov/echallan/config/ChallanConfiguration.java (hunk 1 above)
Comment (rahu-eGov) [reply municipal-services/echallan-services/src/main/java/org/egov/echallan/config/ChallanConfiguration.java]: Done

File: municipal-services/echallan-services/src/main/java/org/egov/echallan/config/ChallanConfiguration.java (hunk 2 above)
Comment (rahu-eGov) [reply municipal-services/echallan-services/src/main/java/org/egov/echallan/config/ChallanConfiguration.java]: Done

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**